docs/quick-starts/bandit-quickstart.md

```diff
@@ -0,0 +1,200 @@
+
+![Add variations step](/img/bandit/bandit-qs-2.png)
+
+### 5. Initialize the SDK
+
+Choose the Eppo SDK that fits in your stack. You'll need to initialize the SDK in your app and create an Eppo client. Here is an example in Java:
+
+```java
+EppoClientConfig config = EppoClientConfig.builder()
+        .apiKey("<api-key>")
+        .assignmentLogger((data) -> System.out.println(data.toString()))
+        .build();
+EppoClient eppoClient = EppoClient.init(config);
+```
+Note, here is where you use the SDK key generated in step 1.
+
+### 6. Embed the Bandit in your code
+
+Once the SDK is initialized, use getStringAssignment() to check what variation a user should see. When requesting an assignment from a flag with a bandit, the set of actions and their attributes are provided as an additional argument.
+
+For example, in the Java SDK, the call may look like:
+```java
+// Flag that has a bandit variation
+String banditTestFlagKey = "bandit-test";
+
+// Subject information--same as for retrieving simple flag or experiment assignments
+String subjectKey = username;
+EppoAttributes subjectAttributes = userAttributes;
+
+// Action set for bandits
+Map<String, EppoAttributes> actionsWithAttributes = Map.of(
+  "dog", new EppoAttributes(Map.of(
+    "legs": EppoValue.of(4),
+    "size": "large"
+  )),
+  "cat", new EppoAttributes(Map.of(
+    "legs": EppoValue.of(4),
+    "size": "medium"
+  )),
+  "bird", new EppoAttributes(Map.of(
+    "legs": EppoValue.of(2),
+    "size": "medium"
+  )),
+  "goldfish", new EppoAttributes(Map.of(
+    "legs": EppoValue.of(0),
+    "size": "small" 
+  ))   
+);
+
+Optional<String> banditAssignment = eppoClient.getStringAssignment(subjectKey, flagKey, subjectAttributes, actionsWithAttributes);
+```
+
+If the actions don't have attributes for the context, you can simply provide a set of actions without attributes:
+```java
+Set<String> actions = Set.of("dog", "cat", "bird", "goldfish");
+
+Optional<String> banditAssignment = eppoClient.getStringAssignment(subjectKey, flagKey, subjectAttributes, actions);
+```
+
+Note that the `getAssignment` methods in Eppo are deterministic, meaning that they will always return the same variant for a given subject (e.g., user) for the life of the bandit.
+
+### 7. Train the Bandit 
+:::note
+Check that you're in the Test environment
+:::
+
+Using the template provided for Bandit Training, click `Add Feature Gate Allocation`. We are going to serve the Bandit to 100% of traffic in the test environment. Click the `Save` button to confirm.
+
+![Add bandit training to the Test environment](/img/bandit/bandit-qs-3.png)
+
+### 8. Turn the Bandit on in Test
+
+To start the Bandit, flip the flag on in the Test environment.
+
+![Turn on the bandit using the toggle](/img/bandit/bandit-qs-4.png)
+
+You should now see assignments coming through the Eppo SDK!
```

Comment:
Maybe we should say you should "soon" see assignments coming through the Eppo SDK, as polling is done every 30 seconds with some random jitter (Up to 5 seconds for Java SDK) as well.